job_executor/model/metadata.py

```diff
@@ -362,39 +323,20 @@ def patch(self, other: "Metadata") -> "Metadata":
             "sensitivityLevel": self.sensitivity_level,
             "populationDescription": other.population_description,
             "subjectFields": [field for field in other.subject_fields],
-            "temporalCoverage": self.temporal_coverage.dict(),
+            "temporalCoverage": self.temporal_coverage.model_dump(),
             "measureVariable": (
-                self.measure_variable.patch(other.measure_variable).dict()
+                self.measure_variable.patch(
+                    other.measure_variable
+                ).model_dump()
             ),
             "identifierVariables": [
                 self.identifier_variables[0]
                 .patch(other.identifier_variables[0])
-                .dict()
+                .model_dump()
```

Comment:
Let's use the aliases, and exclude nones here too for the sake of consistency 🙏🏻